internal/saucecloud/xcuitest.go

```diff
@@ -30,6 +30,11 @@ func (r *XcuitestRunner) RunProject() (int, error) {
 		return exitCode, err
 	}
 
+	if r.DryRun {
+		r.dryRun(r.Project)
```

Comment:
`r.Project` is already in `XcuitestRunner`. I think no need to pass it here and just pick it from `XcuitestRunner`.

Reply:
👍